build/teamcity/internal/release/process/make-and-publish-build-artifacts.sh

```diff
@@ -126,5 +135,13 @@ cat << EOF
 
 Build ID: ${build_name}
 
+The binaries are available at:
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.linux-amd64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.linux-arm64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.darwin-10.9-amd64.tgz
+  https://storage.googleapis.com/cockroach-builds-artifacts-prod/cockroach-$build_name.windows-6.2-amd64.zip
```

Comment:
Let's have these print the location of the files for the location they were placed in.

```suggestion
  https://storage.googleapis.com/$gcs_bucket/cockroach-$build_name.linux-amd64.tgz
  https://storage.googleapis.com/$gcs_bucket/cockroach-$build_name.linux-arm64.tgz
  https://storage.googleapis.com/$gcs_bucket/cockroach-$build_name.darwin-10.9-amd64.tgz
  https://storage.googleapis.com/$gcs_bucket/cockroach-$build_name.windows-6.2-amd64.zip
```